### ISSUE TITLE
Switch to ºF

### ISSUE THREAD

**Issue by Aariq** [cct-datascience/thermal-trends#15]
Produce results in ºF (or just do a transformation on the legend)

**Comment by Aariq** [comment]
Or maybe two legends?  one for ºC and one for ºF?

**Comment by KristinaRiemer** [comment]
Decided to do this mainly because fahrenheit is what land managers will be using...?

**Comment by Aariq** [comment]
Yes, exactly.  I still suspect the journal might require SI units, depending on the journal, but we can convert thresholds.  E.g. 50GDD in ºF is 10GDD ºC
